Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -693,11 +794,12 @@ BOOST_PYTHON_MODULE(rdMolAlign) {
       (python::arg("prbMol"), python::arg("refMol"), python::arg("prbId") = -1,
        python::arg("refId") = -1, python::arg("map") = python::object(),
        python::arg("maxMatches") = 1000000,
+       python::arg("symmetrizeConjugatedTerminalGroups") = false,
```

Comment:
I agree that this should be `true`, but the corresponding entry in the "backwards incompatible changes" part of the release notes should be updated.